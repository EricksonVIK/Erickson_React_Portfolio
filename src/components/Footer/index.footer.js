import React from "react";
import github from "../../assets/images/GitHub-icon.png";
import linkedin from "../../assets/images/linkedin-icon.png";
import email from "../../assets/images/email-icon.png";

function Footer() {
    return (
        <footer>
            <a href="https://github.com/EricksonVIK" target="_blank"><img className="footer-icon" src={`${github}`} alt="github icon"></img></a>
            <a href="https://www.linkedin.com/in/cserickson/" target="_blank"><img className="footer-icon" src={`${linkedin}`} alt="linkedin icon"></img></a>
            <a href="mailto://ericksonvik76@gmail.com"><img className="footer-icon" src={`${email}`} alt="email icon"></img></a>
        </footer>
    )
}

export default Footer;