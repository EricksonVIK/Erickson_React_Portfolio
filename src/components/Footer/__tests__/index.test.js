// cleanup -- removes components
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
// import compnent to be tested
import Footer from "../index.footer";

// ensures no leftover memory to clash
afterEach(cleanup);

describe("Footer component", () => {
  // First Test
  it("renders", () => {
    render(<Footer />);
  });
  // Second Test
  it("matches snapshot DOM node structure", () => {
      const { asFragment } = render(<Footer />);
      expect(asFragment()).toMatchSnapshot();
  });
});