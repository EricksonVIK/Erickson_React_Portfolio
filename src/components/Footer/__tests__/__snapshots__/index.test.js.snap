// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <footer>
    <a
      href="https://github.com/EricksonVIK"
      target="_blank"
    >
      <img
        alt="github icon"
        class="footer-icon"
        src="GitHub-icon.png"
      />
    </a>
    <a
      href="https://www.linkedin.com/in/cserickson/"
      target="_blank"
    >
      <img
        alt="linkedin icon"
        class="footer-icon"
        src="linkedin-icon.png"
      />
    </a>
    <a
      href="mailto://ericksonvik76@gmail.com"
    >
      <img
        alt="email icon"
        class="footer-icon"
        src="email-icon.png"
      />
    </a>
  </footer>
</DocumentFragment>
`;
